NotificationContent/NotificationViewController.swift

```diff
@@ -30,7 +30,10 @@ class NotificationViewController: UIViewController, UNNotificationContentExtensi
                     podcastImage.image = UIImage(data: data)
                     attachment.url.stopAccessingSecurityScopedResource()
                 }
-            } catch {}
+            } catch {
+                // Log the error for debugging purposes
+                print("NotificationViewController: Failed to load podcast image - \(error.localizedDescription)")
```

Comment:
Let's not expose the log and use it for debug only
```suggestion
#if DEBUG
                print("NotificationViewController: Failed to load podcast image - \(error.localizedDescription)")
#endif
```